templates/commands/render_test.go

```diff
@@ -232,6 +232,10 @@ func TestRealRun(t *testing.T) {
 				t.Error(diff)
 			}
 
+			if fg.gotSource != r.source {
+				t.Errorf("fake getter got template source %s but wanted %s", fg.gotSource, r.source)
+			}
```

Comment:
I _think_ Chen might have wanted this:

```suggestion
			if got, want := fg.gotSource, r.source {
				t.Errorf("fake getter got template source %s but wanted %s", got want)
			}
```